src/components/input-style/input-style.scss

```diff
@@ -1,36 +1,61 @@
 @import 'commons';
 
 $m-input-style-base-height: $m-line-height + em;
+$m-input-style-border-radius: 3px;
+$m-searchfield-color: $m-color--grey-lightest;
 
 @mixin input-style-cursor($cursor: default) {
     &,
     .m-input-style__label,
-    .m-input-style__body,
-    .m-input-style__body input,
-    .m-input-style__body textarea {
+    .m-input-style__input,
+    .m-input-style__input input,
+    .m-input-style__input textarea {
         cursor: $cursor;
 
         @content;
     }
 }
 
 .m-input-style {
+    @include input-style-cursor(text);
+
     position: relative;
     display: inline-flex;
     outline: none;
 
-    &__content {
+    &__main {
+        border: 1px solid $m-color--grey-light;
+        border-radius: $m-input-style-border-radius;
+        transition: border-color $m-transition-duration ease-in-out, margin-top $m-transition-duration ease;
+        display: inline-flex;
+        width: 100%;
+
+        .m-searchfield & {
```

Comment:
Est-ce que ça serait possible de déplacer le code relier au css du searchfield a l’intérieur du component searchfield? 

On peut le faire en évitant les /deep/ il s'agit de ne pas scoper le css a l’intérieur comme on a fait dans le component add.ts , C'est l'approche qu'on veut favoriser dans modul a l'avenir.  Ca va permettre de mieux maintenir les composant en isolation

Reply:
Wow je vien de voir ça.

Est-ce que ça demande de la config additionnelle ou ça marche out of the box?  Je cherchais un moyen de pas scoper le CSS de certains composants depuis toujours =D

Reply:
Ouais ça fonctionne, on voulait l'activer pour tous les composantes vu qu'on utilise déja un standard BEM et que le style guide de vue suggère de ne pas scoper dans une librairie de composante (https://vuejs.org/v2/style-guide/#Component-style-scoping-essential ) Mais ça créé certaines régressions (nottament dans m-link) donc a utiliser avec parcimonie dans modUL et surtout pas dans Brio 👀

Reply:
Ouais mais dans brio on a quand même eu une couple de cas :)

Reply:
J'ai quand même partagé aux gens de brio, merci!

Reply:
C'est franchement une bonne nouvelle! ;)